Lib/fontTools/misc/transform.py

```diff
@@ -423,6 +423,39 @@ class DecomposedTransform:
     tCenterX: float = 0
     tCenterY: float = 0
 
+    def __eq__(self, other):
```

Comment:
this is a dataclass so it already implements `__eq__` and `__ne__` by default, no? eq=True by default
why do you need to override them?

`__bool__` probably is not implemented for you so you may want to add that manually, but the other two should already be there.